arviz/tests/test_plots.py

```diff
@@ -34,14 +33,57 @@
 np.random.seed(0)
 
 
+def create_model(seed=10):
+    """Create model with fake data."""
+    np.random.seed(seed)
+    nchains = 4
+    ndraws = 500
+    data = {
+        "J": 8,
+        "y": np.array([28.0, 8.0, -3.0, 7.0, -1.0, 1.0, 18.0, 12.0]),
+        "sigma": np.array([15.0, 10.0, 16.0, 11.0, 9.0, 11.0, 10.0, 18.0]),
+    }
+    posterior = {
+        "mu": np.random.randn(nchains, ndraws),
+        "tau": abs(np.random.randn(nchains, ndraws)),
+        "eta": np.random.randn(nchains, ndraws, data["J"]),
+        "theta": np.random.randn(nchains, ndraws, data["J"]),
+    }
+    posterior_predictive = {"y": np.random.randn(nchains, ndraws, len(data["y"]))}
+    sample_stats = {
+        "energy": np.random.randn(nchains, ndraws),
+        "diverging": np.random.randn(nchains, ndraws) > 0.90,
+        "log_likelihood": np.random.randn(nchains, ndraws, data["J"]),
+    }
+    prior = {
+        "mu": np.random.randn(nchains, ndraws) / 2,
+        "tau": abs(np.random.randn(nchains, ndraws)) / 2,
+        "eta": np.random.randn(nchains, ndraws, data["J"]) / 2,
+        "theta": np.random.randn(nchains, ndraws, data["J"]) / 2,
+    }
+    prior_predictive = {"y": np.random.randn(nchains, ndraws, len(data["y"])) / 2}
+    sample_stats_prior = {
+        "energy": np.random.randn(nchains, ndraws),
+        "diverging": (np.random.randn(nchains, ndraws) > 0.95).astype(int),
+    }
+    model = from_dict(
+        posterior=posterior,
+        posterior_predictive=posterior_predictive,
+        sample_stats=sample_stats,
+        prior=prior,
+        prior_predictive=prior_predictive,
+        sample_stats_prior=sample_stats_prior,
+        observed_data={"y": data["y"]},
+        dims={"y": ["obs_dim"]},
+    )
+    return model
+
+
 @pytest.fixture(scope="module")
-def models(eight_schools_params):
+def models():
     class Models:
-        models = load_cached_models(eight_schools_params, draws=500, chains=2)
-        pymc3_model, pymc3_fit = models["pymc3"]
-        stan_model, stan_fit = models["pystan"]
-        emcee_fit = models["emcee"]
-        pyro_fit = models["pyro"]
+        model_1 = create_model(seed=10)

```

Comment:
Why two models?

Reply:
Some plots test multiple models, and this way, if dev saves the plot, the models won't overlap.